src/Roslyn.Diagnostics.Analyzers/Core/Roslyn.Diagnostics.Analyzers.csproj

```diff
@@ -7,6 +7,7 @@
       Restore would conclude that there is a cyclic dependency between us and the Roslyn.Diagnostics.Analyzers package.
     -->
     <PackageId>*$(MSBuildProjectFullPath)*</PackageId>
+    <MicrosoftCodeAnalysisVersion>2.8.2</MicrosoftCodeAnalysisVersion>
```

Comment:
This will lead to us producing 2.6.x version of analyzer package, which will not work against 2.6.x version of Microsoft.CodeAnalysis packages. This is against the versioning scheme used by the repo since inception. I would recommend either creating a separate branch for this change or using the features/dataflow branch which is already on 2.9.x bits of Microsoft.CodeAnalysis.

Reply:
This analyzer package is only used for Roslyn.sln, where the master branch is already on 3.x.

Reply:
Yes, but the version of generated package will still be 2.6.x on master branch. One other option is to override the Version property for this project and corresponding NuGet package so it generates 2.8.x in master branch to maintain consistency.

Reply:
Master has now moved to Microsoft.CodeAnalysis `2.9.0`. This change can be reverted.